java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -0,0 +1,67 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * List Commands interface for both standalone and cluster clients.
```

Comment:
I feel like each interface is documented differently, some have `for both standalone and cluster clients` some have `for all server requests` some have both and some have none, I think we should stay consistent.

Reply:
Added a ticket in our backlog to update all of them at once.